### PR TITLE
start awesomifying sass

### PR DESCRIPTION
**Pull request by catherine-jones**
- makes break points for kpi consistent now only 3 layouts across various breakpoints
- refactors media queries to be mobile first
- replaces neat with bootstrap
- refactors code to be cleaner and less brittle

### CODE REVIEW

**Comment by catherine-jones** [comment]
- [ ] IE9
- [ ] IE10
- [ ] IE11
- [ ] IE13
- [ ] Edge
- [ ] Chrome/responsive
- [ ] Safari
- [ ] Firefox
- [ ] iphone 5
- [ ] iphone 6
- [ ] iphone 7
- [ ] ipad air
- [ ] ipad mini
- [ ] ipad pro
- [ ] Samsung Galaxy S6
- [ ] Screen reader
- [ ] WAVE

File: app/views/dashboards/_kpi-widget.html.erb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <div data-id="<%= widget.id %>" class="col-xs-12 col-md-3">
   <div class="widget <%= widget.name_slug %> has-data-<%= widget.has_data? %> <%= widget.style %>">
-    <div id="<%= widget.name_slug %>" data-id="<%= widget.id %>" class="widget__title">
+    <div id="<%= widget.name_slug %>" data-id="<%= widget.id %>" class="widget__title col-xs-8">
```
Comment (elisechant) [inline app/views/dashboards/_kpi-widget.html.erb]: You need to include a row like outer-container. This was the same issue that caused bugs with the Neat grid.  Make sure you read the documentation.

**Comment by elisechant** [comment]
Can you summarise the change?

**Comment by catherine-jones** [comment]
@elisechant changes are summarised up the top

**Pull request closed without merging**